data/transactions/verify/txn.go

```diff
@@ -448,3 +456,394 @@ func (w *worksetBuilder) next() (txnGroups [][]transactions.SignedTxn) {
 func (w *worksetBuilder) completed() bool {
 	return w.idx >= len(w.payset)
 }
+
+// UnverifiedElement is the element passed the Stream verifier
+// Context is a reference associated with the txn group which is passed
+// with the result
+type UnverifiedElement struct {
+	TxnGroup []transactions.SignedTxn
+	Context  interface{}
```

Comment:
The internal package `verify` does not care about the content of this element. The `data` package defines and uses it, and also imports `verify`. `verify` cannot know about it since it will introduce a circular dependency.